src/services/ArrayInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
+namespace SwitchViews\services;
```

Comment:
It is ideal to that namespace names should start with Capital Letter. In this case **services** should be **Services**